velox/dwio/parquet/reader/PageReader.h

```diff
@@ -321,7 +336,7 @@ class PageReader {
 
   memory::MemoryPool& pool_;
 
-  std::unique_ptr<dwio::common::SeekableInputStream> inputStream_;
+  std::shared_ptr<dwio::common::SeekableInputStream> inputStream_;
```

Comment:
This is not thread safe and better not to be shared.  Maybe we should take in raw pointer instead of `shared_ptr` in `ThriftStreamingTransport`.